integration_tests/pages/additionalSupport.ts

```diff
@@ -8,7 +8,7 @@ export default class AdditionalSupportPage extends Page {
   additionalSupportRequired = (): PageElement => cy.get('[data-test=support-required-yes]')
 
   enterOtherSupportDetails = (details: string): void => {
-    cy.get('#otherSupportDetails').clear().type(details)
```

Comment:
Updated lint rules for Cypress complained about chaining after `.clear()`. I just removed it because it's currently unnecessary: the inputs _should_ generally be empty - something's gone wrong if they're not. (May need to revisit this in some specific cases - e.g. where things may be pre-populated when testing update journey...)